faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
```

Comment:
1.  +1 on querying Amy for URLs of upcoming workshops.
2.  We should fetch the actual info of those workshops from their 
websites (since instructors, rooms, etc. can change at the last minute).
3.  We need to archive info about past workshops in the `site` repo - 
can't be sure their repos will still be there in a year.